src/views/cart/index.hbs

```diff
@@ -0,0 +1,68 @@
+<div class="cart w-100 mw-xl">
+    <div class="cart-list  ">
+        <p class="fs-3 fw-medium">Card</p>
+        <table class="table table-hover">
+            <thead>
+                <tr>
+                    <th scope="col">Product detail</th>
+                    <th scope="col">Quantity</th>
+                    <th scope="col">total</th>
+                    <th scope="col">remove</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr class="row-product text-center">
+                    <th scope="row" class="align-middle">
+                        <div class="product-cart height-100 d-flex  align-items-center">
+                            <div>
+                                <img class="product-cart__img"
+                                    src="https://product.hstatic.net/1000075078/product/1675355354_bg-tch-sua-da-no_4fbf208885ed464ab4b5e145336d42a2_large.jpg"
+                                    alt="product image">
+                            </div>
+                            <div class="cart-product">
+                                <p class="cart-name">Hamburgers And Chips Meal</p>
+                                <div class="cart-topping">
+                                    <p>+Trân châu trắng</p>
+                                    <p>+Trân châu đen</p>
+                                </div>
+                                <p class="cart-price mb-0">+33.000 đ</p>
+                            </div>
+                        </div>
+                    </th>
+                    <td class="align-middle">
+                        <div class="form-outline">
+                            <input type="number" id="typeNumber" class="form-control number-input text-center" />
+                        </div>
+                    </td>
+                    <td class="align-middle text-center">
+                        <p class="cart-price mb-0">333.000 đ</p>
+                    </td>
+                    <td class="align-middle text-center">
+                        <i class="fa-sharp fa-solid fa-trash"></i>
+                    </td>
+                </tr>
+
+            </tbody>
+        </table>
+    </div>
+    <div class="cart-bill">
+        <div class="cart-order">
+            <div class="mw-xs">
+                <div class="order-title fs-4">ORDER SUMMARY</div>
+                <div class="d-flex justify-content-between align-items-center pb-14">
+                    <div>subtotal</div>
+                    <div>1435$</div>
```

Comment:
phần này chưa có tính toán tự động e nhỉ?

Reply:
Dạ phần UI cho trang cart em xây UI trước nên còn fix cứng á chị